src/cmake/STIRConfig.h.in

```diff
@@ -81,6 +81,8 @@ namespace stir {
 
 #cmakedefine HAVE_SYSTEM_GETOPT
 
+#define STIR_V4_PROJDATA_MATHS
```

Comment:
I don't really fancy doing this. we'll end up with having a define for every feature, and it'll be a mess very soon (as we'll have it for some but not others). I think we should just release STIR very often...

Reply:
I can remove, but then people couldn't have old STIR with new SIRF. Fine by me.

Reply:
we get into a mess otherwise anyway on the SIRF side. If it isn't in STIR, it'll have to be in SIRF, so we then get into duplication etc.

So, I suggest you merge this, do a PR on Array, and release 4.1 beta, all by Sunday ...

Can you put a line in the 4.1 release notes?